tock/projects/models.py

```diff
@@ -23,6 +23,8 @@ class Project(models.Model):
     name = models.CharField(max_length=200)
     iaa = models.CharField(max_length=200, blank=True)
     agency = models.ForeignKey(Agency)
+    description = models.TextField(blank=True, null=True)
+    billable = models.BooleanField(default=False)
```

Comment:
I suppose that's right. Perhaps I'm anticipating situations where it's billable through other means? But hard to know what that might be.